pyjade/compiler.py

```diff
@@ -377,6 +379,9 @@ def visitAttributes(self, attrs):
 
         if temp_attrs: self.visitDynamicAttributes(temp_attrs)
 
+    def visitAndAttributes(self, andattrs):
+        self.buffer('{%% for k in %s %%} {{ k }}="{{ k[v] }}"{%% endfor %%}' % andattrs)
```

Comment:
I run a test with the following jade code but it cannot render the values properly.
```
- set a = {'attributes': {'foo': 'bar', 'hello': 'world'}}
input&attributes(a.attributes)
```

it compiled to: (jinja2)
```
{%  set a = {'attributes': {'foo': 'bar', 'hello': 'world'}} %}
<input{% for k in a.attributes %} {{ k }}="{{ k[v] }}"{% endfor %}/>
```

the jinja2 template is rendered as:
```

<input foo="" hello=""/>
```

maybe change it to  ```self.buffer('{%% for k,v in %s.iteritems() %%} {{ k }}="{{ v }}"{%% endfor %%}' % andattrs)```?